whisper/src/lib.rs

```diff
@@ -713,14 +713,18 @@ fn file_merge<F1: Read + Seek, F2: Read + Write + Seek>(fh_src: &mut F1, fh_dst:
     Ok(())
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct DiffPoint {
     pub interval: u32,
     pub value1: Option<f64>,
     pub value2: Option<f64>,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct DiffArchive {
+    #[serde(rename = "archive")]
```

Comment:
why in json you rename it? it admix meaning a bit

Reply:
Because json defined is defined in bin/whisper-diff.py and not related to lib directly.

Reply:
but the struct is in lib...
and it's purpose is, in this case, fishily...

Reply:
Indeed. Right now we try to implement all cli to find the usage. Some API inside the lib was done only for cli and will be refactored and moved to another space/create/etc later.